org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "org.arrl.trustedqsl",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
+	"--filesystem=home"
```

Comment:
this is a broad filesystem permission.

Reply:
TrustedQSL stores configuration data in $HOME/.tqsl and can be used to sign log files from other logging applications.  I don't think full filesystem access is needed, but the way this is used (primarily as a utility for loggers to sign and upload), home access is needed.

Reply:
then you can use `--persist` instead.

ideally it would use xdg standard locations.

Reply:
I'll try with --persist.

Reply:
`--persist=.tqsl` should work fine.